85.md

```diff
@@ -0,0 +1,54 @@
+NIP-85
+======
+
+Reviews
+-------
+
+`draft` `optional` `author:staab`
+
+Kind `1986` MAY be used to leave a review with optional ratings.
+
+The `content` field SHOULD include a human-readable review.
+
+The review event MUST include one or more `e`, `a`, `p`, `t`, or `r` tags indicating
+the object of the review. This allows for reviews of events, people, topics, and relays
+or external resources. As with NIP-01, a relay hint SHOULD be included when using `e` and
+`p` tags.
+
+Tags MAY contain additional `rating` tags each of which should have a `value` between 0 and 1,
+and an optional `mark` specifying an attribute of the thing being reviewed.
+
+Tags MAY contain NIP 32 labels to help categorize what is being reviewed.
+
+Examples
+--------
+
+A relay review with rating metadata.
+
+```json
+{
+  "kind": 1986,
+  "content": "This relay is fast!",
+  "tags": [
+    ["L", "review"],
+    ["l", "review/relay", "review"],
+    ["rating", "0.8"],
+    ["rating", "0.2", "smell"],
+    ["rating", "1", "service"],
+    ["r", <relay_url>]
```

Comment:
I think we need to stop overloading `r` to mean both relay and resource. NIP-65 is a mistake in that area and I think we need to reverse that rather than extend this problem.

Not to open another can of worms here (but I will I suppose) but here's another good reason for relays to have a pubkey and this would simply be just another p-tag.

Reply:
I agree, although I think we should probably do the opposite and refer to relays as `r` and urls as `u` or something. Referring to relays via p tag is interesting, but would require clients to do a lot more work, when a url is already ideal (it's a "universal resource locator"). I think it's probably better to use the url anyway since you can't de-refence a pubkey to a url if you don't already know where to look.